src/lib/components/ProgressBar.svelte

```diff
@@ -14,7 +14,7 @@
 	});
 
 	$: $progress = player.progressPercent;
-  $: console.log(player.progressPercent, player.score, player.scoreRequired)
+	$: console.log(player.progressPercent, player.score, player.scoreRequired);
```

Comment:
```suggestion
```

Reply:
I wanted to leave this in to prevent rebasing. Then ES lint complained on ever other PR so there's gonna be rebasing no matter what.